views/tables/ParticipantsTable/index.jsx

```diff
@@ -135,14 +136,19 @@ const ParticipantsTable = (props) => {
           />
         )
       case 'study':
+        const participantProperty = participant[property]
+        const tooltipTitle = SITE_NAMES[participantProperty] || participantProperty
+        
         return (
-          <Typography
-            component={Link}
-            to={routes.studyDashboard(participant.study)}
-            sx={{ textDecoration: 'none', color: 'text.primary' }}
-          >
-            {participant[property]}
-          </Typography>
+          <Tooltip title={tooltipTitle} placement="right">
+            <Typography
+              component={Link}
+              to={routes.studyDashboard(participant.study)}
+              sx={{ textDecoration: 'none', color: 'text.primary' }}
+            >
+              {participant[property]}
```

Comment:
should this be `tooltipTitle`?

Reply:
This should have been `participantProperty`. In this context `participantProperty` is the site code (BM) and the `tooltipTitle` is its full name (Birmingham). We are preserving the site code on the participant table and showing the full site name only on hover.